app/src/main/res/layout/profile_progress_header.xml

```diff
@@ -175,7 +176,7 @@
       android:fontFamily="sans-serif-medium"
       android:minHeight="48dp"
       android:text="@string/recently_played_stories"
-      android:textColor="@color/component_color_profile_progress_activity_recently_played_stories_text_color"
+      android:textColor="@color/component_color_shared_content_item_text_view_color"
```

Comment:
Remove `component_color_shared_content_item_text_view_color` and use already existing color as suggested earlier.

Reply:
All that to other textviews too in this file and to other similar header files.